compiler/src/main/java/ca/mcscert/jpipe/model/elements/JustificationModel.java

```diff
@@ -139,4 +166,16 @@ public String toString() {
         return sb.toString();
     }
 
+    public JustificationElement getDirectReplacementOf(JustificationElement elem) {
+        return repTable.getParent(elem);
+    }
+
+    public JustificationElement getOriginalOf(JustificationElement elem) {
```

Comment:
should be eventually deleted if never used (after error handling implementation)